Bento/Adapters/CollectionViewAdapter.swift

```diff
@@ -114,11 +114,15 @@ open class CollectionViewAdapterBase<SectionID: Hashable, ItemID: Hashable>
 
     @objc(collectionView:canPerformAction:forItemAtIndexPath:withSender:)
     open func collectionView(_ collectionView: UICollectionView, canPerformAction action: Selector, forItemAt indexPath: IndexPath?, withSender sender: Any?) -> Bool {
-        guard let indexPath = indexPath,
-            let component = sections[indexPath.section].items[indexPath.row].component(as: MenuItemsResponding.self) else {
-            return false
+        guard let indexPath = indexPath else { return false }
+        return self.collectionView(collectionView, canPerformAction: action, forItemAt: indexPath, withSender: sender)
+    }
+    
+    open func collectionView(_ collectionView: UICollectionView, canPerformAction action: Selector, forItemAt indexPath: IndexPath, withSender sender: Any?) -> Bool {
```

Comment:
Why not just change `IndexPath?` to `IndexPath` and remove the guard for it, instead of adding a new method?

Reply:
Ok, looks like `UIKit` can pass `null` to params that are annotated as non-null 🤦‍♀️

Reply:
I saw that @sergdort made a commit specifically for nil IndexPaths last November and, as such, I chose to maintain that same logic